darwin/Platform.c

```diff
@@ -18,6 +18,7 @@ in the source distribution for its full text.
 #include <net/if_types.h>
 #include <net/route.h>
 #include <sys/socket.h>
+#include <mach/mach_init.h>
```

Comment:
Why is this here, when it's not used through new code? Or is was this missing before; and by which code?